Necrosis/XML/Attributes.lua

```diff
@@ -312,11 +332,18 @@ function Necrosis:StoneAttribute(Steed)
 	NecrosisSpellTimerButton:SetAttribute("macrotext", "/focus")
 	NecrosisSpellTimerButton:SetAttribute("type2", "item")
 	NecrosisSpellTimerButton:SetAttribute("item", self.Translation.Item.Hearthstone)
+
+	-- if the 'Ritual of Souls' spell is known, then associate it to the hearthstone shift-click.
```

Comment:
`Ritual of Souls` isn't a part of classic, Doesn't exist in spell ID chart. Suggest removing